var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -448,6 +449,7 @@ class Trilinos(CMakePackage, CudaPackage, ROCmPackage):
         sha256="063a38f402439fa39fd8d57315a321e6510adcd04aec5400a88e744aaa60bc8e",
         when="@13.0.0:13.0.1 +teko gotype=long",
     )
+    patch("slu-8.patch", when="@13.4.0 +superlu-dist")
```

Comment:
Can you add the provenance for the patch file? Better yet, if the patch is a standalone PR/commit, can you delete the patch file and replace with something like:
```
    patch(
        "https://patch-diff.githubusercontent.com/raw/trilinos/Trilinos/pull/10545.patch?full_index=1",
        sha256="blah",
        when="@13.4.0 +superlu-dist"
    )
```